frontend/revive-gallery-app/src/components/home/Navbar.js

```diff
@@ -24,53 +24,33 @@ const Navbar = () => {
         <li>
           <Link to="/">Home</Link>
         </li>
-
         <li>
           <Link to="/products">Products</Link>
         </li>
-
         <li className="dropdown">
           <span onClick={toggleDropdown}>Categories</span>
           {isDropdownOpen && (
             <ul className="dropdown-content">
               {categories.map((category) => (
                 <li key={category.name}>
-                  <Link to={`/products?categories=${category.name}`}>{category.name}</Link>
+                  <Link to={`/products?categories=${category.name}`}>
+                    {category.name}
+                  </Link>
                 </li>
               ))}
             </ul>
           )}
         </li>
-
         {user && token ? (
+          <>
             <li>
               <Link to="/addproduct">Add Product</Link>
             </li>
-          ) : null}
-
-        {user && token ? (
-          <li className="dropdown">
-            <span onClick={toggleDropdown}>User Profile</span>
-            {isDropdownOpen && (
-              <ul className="dropdown-content">
-                <li>
-                  <Link to="/user/trustworthiness">Trustworthiness Score</Link>
-                </li>
-                <li>
-                  <Link to="/user/wishlist">Wishlist</Link>
-                </li>
-                <li>
-                  <Link to="/user/cart">Cart</Link>
-                </li>
-              </ul>
-            )}
-          </li>
-        ): 
-        (<li>
-          <Link to="/login">Sign in/</Link>
-          <Link to="/register">Sign up</Link>
-        </li>)
-        }
```

Comment:
@bkarakala I think we can leave these as they are; we do need to remove them. Even though those links do not provide meaningful data as of now, they do not break the code.